fiftyone/utils/cityscapes.py

```
@@ -288,6 +303,26 @@ def _export_split(
     dataset.delete()
 
 
+def _merge_extra_images(images_extra_zip_path, images_dir, scratch_dir):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_merge_extra_images`

Add return type annotation: `None`

(ANN202)